invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/components/requests.js

```diff
@@ -139,21 +141,39 @@ export function RequestsResultsGridItemTemplate({ result, index }) {
 }
 
 export function RequestsResultsItemTemplate({ result, index }) {
+  const createdDate = new Date(result.created);
+  const differenceInDays = timestampToRelativeTime(createdDate.toISOString());
   return (
     <Item key={index}>
       <Item.Content>
-        <Item.Header href={`/requests/${result.id}`}>
-          {result.metadata.title}
-          {result.metadata.type && (
-            <span style={{ marginLeft: "5px" }}>
-              <Label size="small">{result.metadata.type}</Label>
+        <Item.Header>
+          {result.type && (
+            <span className="mr-5">
+              <Label size="large">{result.type}</Label>
             </span>
           )}
+          <a href={`/requests/${result.id}`}>{result.title}</a>
         </Item.Header>
-        <Item.Meta>
-          <div
-            dangerouslySetInnerHTML={{ __html: result.metadata.description }}
-          />
+
+        <Item.Meta className="mt-10">
+          <span className="mr-15">
+            {/* TODO: Replace by resolved user */}
+            {/* {i18next.t(`opened {{difference}} by {{user}}`, {
+              difference: differenceInDays,
+              user: result.created_by.user,
+            })} */}
+            {`opened ${differenceInDays} by you`}
+          </span>
+          {result.receiver.community && (
+            <>
+              <Icon className="default-margin" name="users" />
+              <span className="ml-5">
+                {/* TODO: Replace by resolved receiver */}
```

Comment:
Maybe open an issue instead of adding the TODOs so we don't forget and the code is clean?

Reply:
I already opened an issue see https://github.com/inveniosoftware/invenio-app-rdm/issues/1218